app/projects/results-summary-component/page.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { Hanken_Grotesk } from 'next/font/google';
+import { Wrapper, Title, Card } from './styles';
+import Button from './components/Button';
+import StatCard from './components/StatCard';
+import { BrainSVG, ThunderSVG, VerbalSVG, VisualSVG } from './icons';
+
+export const HankenGrotesk = Hanken_Grotesk({
```

Comment:
![image](https://github.com/CamiStefano/my-portfolio/assets/5130752/9a0aad51-b2ef-4101-aa7f-2473b861b8fe)

Fix the Vercel preview error